setup.py

```diff
@@ -22,10 +17,11 @@
         'Programming Language :: Python :: 3.7',
         'License :: OSI Approved :: BSD License',
     ],
-    packages=['beastling', 'beastling.clocks', 'beastling.fileio', 'beastling.models', 'beastling.treepriors'],
+    packages=find_packages(),
```

Comment:
Oh, wonderful!  I always thought it was seriously broken that we had to specify these manually one by one.